src/components/details-view/details-view.js

```diff
@@ -198,6 +199,12 @@ export default class DetailsView extends Component {
 
     let historyState = router.history.location.state;
 
+    const countFilters = (params) => {
+      const { q, filter = {} } = params;
+      const appliedFilters = [q, ...Object.values(filter)].filter(val => !!val);
```

Comment:
We should use `let` for these. In the rest of the app we use `const` for top-level constants and `let` for anything else that's scoped to a block.

Reply:
Man, I always forget this haha. You're right.